content/docs/user-guide/how-to/add-output-to-stage.md

```diff
@@ -0,0 +1,69 @@
+# Add Output to Stage
+
+There are situations where you might have forgotten to specify one of the
+<abbr>outputs</abbr> for a stage while creating it, and then later want to add
+an output to that stage.
+
+Running a stage to specify additional output is an expensive step, as it
+requires time and resources to execute the command. Follow the steps listed
+below to add an output to the stage without running it again.
+
+Let's first create a stage for the example <abbr>project</abbr>:
+
+```dvc
+$ dvc run -n prepare \
+          -d src/prepare.py \
+          -o data/train \
+          python src/prepare.py
+```
+
+You can check stage entry in `dvc.yaml`:
+
+```dvc
+$ cat dvc.yaml
+stages:
+  prepare:
+    cmd: python src/prepare.py
+    deps:
+    - src/prepare.py
+    outs:
+    - data/train
+```
+
+Now to add an output to the `prepare` stage, you need to use `dvc run` with the
```

Comment:
```suggestion
To add another output to the `prepare` stage, you need to use `dvc run` with the
```
> May need formatting.

Reply:
Nice writing BTW, thanks!

But again, this is very `run`-centric. Let's explain what should happen to the dvc.yaml file first and then add the `run -f --no-exec` command as an alternative (note) to manual editing.

Reply:
The emphasis (in the update) is on editing `dvc.yaml` now I think. Also `dvc run` is moved to a block quote.